dvc/stage.py

```diff
@@ -557,7 +560,7 @@ def create(repo, accompany_outs=False, **kwargs):
         else:
             path = os.path.abspath(fname)
 
-        Stage._check_stage_path(repo, wdir)
+        Stage._check_stage_path(repo, wdir, is_wdir=kwargs.get("wdir"))
```

Comment:
This will only throw error for `dvc.run()` cases where `wdir` is explicitly set.